cura/Settings/MachineManager.py

```diff
@@ -1496,3 +783,268 @@ def setSettingForAllExtruders(self, setting_name: str, property_name: str, prope
         for key, extruder in self._global_container_stack.extruders.items():
             container = extruder.userChanges
             container.setProperty(setting_name, property_name, property_value)
+
+    #
+    # New
+    #
+
+    # We not fetch it from _current_root_material_id, but later we can get it from somewhere else
```

Comment:
This function documentation makes no sense at all to me.

Reply:
Removed in bb04afa2a